surface/include/pcl/surface/impl/convex_hull.hpp

```diff
@@ -215,15 +214,15 @@ pcl::ConvexHull<PointInT>::performReconstruction2D (PointCloud &hull, std::vecto
 
   int num_vertices = qh num_vertices;
   hull.points.resize (num_vertices);
-  memset (&hull.points[0], hull.size (), sizeof (PointInT));
+  hull.points.clear();
```

Comment:
Clear needs to come before resize

Reply:
Also, hull is of type pointcloud, so `hull.resize()` and `hull.clear()` can be used.
But is it necessary to call first `clear()` and then `resize()` - isn't just sufficient with `resize()`?

Reply:
Ahh, I thought the prs #4218 and #4221 etc was merged already...

Reply:
The original intent was to clear and resize the `hull.points` vector, not the `hull` vector itself, right?

Reply:
If hull is a point cloud, the functions that operate on `hull.points` should be available for hull itself as per the API of point cloud